src/room/InCallView.tsx

```diff
@@ -470,6 +548,56 @@ export const InCallView: FC<InCallViewProps> = ({
     });
   }, [localParticipant, isScreenShareEnabled]);
 
+  const toggleRaisedHand = useCallback(async () => {
+    try {
+      if (isHandRaised) {
+        try {
+          if (reactionId) {
+            await client.redactEvent(rtcSession.room.roomId, reactionId);
+            setReactionId(null);
+            setRaisedHands(raisedHands.filter((id) => id !== userId));
+            logger.debug("Redacted reaction event");
+          }
+        } catch (e) {
+          logger.error("Failed to redact reaction event", e);
+        }
+      } else {
+        const m = memberships.filter((m) => m.sender === userId);
+        const eventId = m[0].eventId!;
```

Comment:
the membership event itself does not contain the eventId.
This would indeed be the correct fix (adding a getter to the `CallMembership` class.) As a stop gap we can access the private parentEvent property.
We wont merge it like this however but it would help testing the netlify build.
```suggestion
        const eventId = (
          m[0] as unknown as { parentEvent: MatrixEvent }
        ).parentEvent.getId()!;

```

Reply:
I just created a matrix-js-sdk PR adding this property to the CallMembership class: https://github.com/matrix-org/matrix-js-sdk/pull/4395

Reply:
Updated the package.json to reference the JS SDK PR merged commit.

(embarrassed that I just realised I locally modified `CallMembership.ts` under `node_modules` thinking it was part of the EC codebase🤦...)

Reply:
I resolve this conversation since the proper fix utilizing the js-sdk patch has been implemented.